README.md

```diff
@@ -83,7 +83,7 @@ Note that non-default networks require extra [firewall setup](https://cloud.goog
   [nfs-client](https://github.com/kubernetes-incubator/external-storage/tree/master/nfs-client)
   external provisioner can be used to provide similar functionality for
   Kubernetes clusters.
-* FsGroup feature: [CSIVolumeFSGroupPolicy](https://kubernetes-csi.github.io/docs/csi-driver-object.html) is a Kubernetes feature in Beta is 1.20, which allows CSI drivers to opt into FSGroup policies. Filestore CSI driver plans to support it in near future. As a workaround, until the feature is available, see user-guide [here](docs/kubernetes/fsgroup.md) on how to apply fsgroup to volumes backed by filestore instances.
+* FsGroup feature: [CSIVolumeFSGroupPolicy](https://kubernetes-csi.github.io/docs/csi-driver-object.html) is a Kubernetes feature in Beta is 1.20, which allows CSI drivers to opt into FSGroup policies. stable-master [overlay](deploy/kubernetes/overlays/stable-master) of Filestore CSI driver now supports this. See user-guide [here](docs/kubernetes/fsgroup.md) on how to apply fsgroup to volumes backed by filestore instances.
```

Comment:
"The stable-master overlay..." and "See the user-guide..." ?

Also maybe add a pointer to the workaround for pre-1.19/feature disabled clusters?

Reply:
Done